dockerfiles/ci/release-build-base/Dockerfile

```diff
@@ -130,4 +130,14 @@ RUN curl -sSL https://github.com/pingcap/tiflash/raw/${TIFLASH_TAG}/release-cent
     source ./install_ccache.sh && \
     install_ccache ${CCACHE_VERSION}
 
-
+# Git, update to 2.40.2 to fix the security issue
```

Comment:
please release with renovate anchor comment to make it can be auto pushing to upgrade to new branch with bot.